.github/actions/build-manager-image/action.yaml

```diff
@@ -0,0 +1,60 @@
+name: 'Build Istio Manager Image'
+description: 'Builds Istio manager image'
+inputs:
+  operator-image-name:
+    description: 'Operator image name used for creation of Istio manager image'
+    required: true
+  platforms:
```

Comment:
After dropping ARM build idea, can this input be removed?

Reply:
Yes, probably if we agree on, that we don't want to support it, I'd remove it together with QEMU.